.gitattributes

```diff
@@ -0,0 +1,9 @@
+.editorconfig          export-ignore
+.gitattributes         export-ignore
+/.github/              export-ignore
+.gitignore             export-ignore
+/docs/                 export-ignore
```

Comment:
If we exclude `docs`, shouldn't we also exclude any `.md` files in the root, other than `LICENSE.md` and `README.md`? At the moment, that's only `CONTRIBUTING.md`, but a wildcard exclusion with those two exceptions might work best.